src/dataflow/components/dataflow-program.tsx

```diff
@@ -118,16 +118,16 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
     return (
       <div className="dataflow-program-container">
         {this.isRunning() && <div className="running-indicator" />}
-        <DataflowProgramTopbar
-            onRunProgramClick={this.runProgram}
-            onStopProgramClick={this.stopProgram}
-            onProgramTimeSelectClick={this.setProgramRunTime}
-            programRunTimes={ProgramRunTimes}
-            programDefaultRunTime={this.props.programRunTime || DEFAULT_PROGRAM_TIME}
-            isRunEnabled={this.isReady()}
-            runningProgram={this.isRunning() && !this.props.readOnly}
-            readOnly={this.props.readOnly || !this.isReady()}
-        />
+        {!this.isComplete() && <DataflowProgramTopbar
+          onRunProgramClick={this.runProgram}
+          onStopProgramClick={this.stopProgram}
+          onProgramTimeSelectClick={this.setProgramRunTime}
+          programRunTimes={ProgramRunTimes}
+          programDefaultRunTime={this.props.programRunTime || DEFAULT_PROGRAM_TIME}
+          isRunEnabled={this.isReady()}
+          runningProgram={this.isRunning() && !this.props.readOnly}
+          readOnly={this.props.readOnly || !this.isReady()}
+        />}
```

Comment:
One side effect of hiding the topbar is that we now have excess vertical space at the bottom of the tile when the topbar is hidden.  It's coming from `toolbar-editor-container` not taking up enough space.  It uses a `calc` in the CSS to determine the height (which takes the topbar height into consideration).  Maybe there is an easier way to do that (some of that CSS probably hasn't been looked at for a few weeks and might be based on previous requirements/stories that are now moot).